src/components/PopupResult/PopupResult.jsx

```diff
@@ -77,29 +67,26 @@ const PopupResult = (props) => {
           </div>
           <div className="popupResultDetails">
             <div className="detailRow">
-              <label>Title: </label>
+              <label htmlFor="title">Title: </label>
```

Comment:
the span lacks an ID for this `<label>` to be tied to. Also, labels won't work for spans, but [aria-labeldby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) will work. Probably should change it from a `<label>` to a styled span, as well, but for the accessibility side:

```suggestion
              <label id="title">Title: </label>
              <span aria-labelledby="title">{props.result.id}</span>
```

Reply:
good callout, this was an attempt to stop the a11y linter from yelling at me without actually taking correct action. Label has now been changed to a span and styled correctly and the appropriate aria-labeldby has been added to show link between the two